gcc/rust/lex/rust-lex.cc

```diff
@@ -240,7 +240,7 @@ Lexer::dump_and_skip (int n)
 				     + std::string (tok->get_type_hint_str ()))
 				  : "")
 	      << " ";
-	  out << Linemap::location_to_string (loc) << " ";
+	  out << Linemap::location_to_string (loc) << '\n';
```

Comment:
is `std::endl` better for this?

Reply:
`std::endl` [will flush the buffer](https://en.cppreference.com/w/cpp/io/manip/endl) AFAIK we don't need to ensure the output buffer is flushed at this point and we should keep the chosen flushing strategy (IIRC it is linemode by default, so it should not change that much in most situations).